lib/ain-cpp-imports/src/bridge.rs

```diff
@@ -29,17 +29,17 @@ pub mod ffi {
 
         fn getChainId() -> u64;
         fn isMining() -> bool;
-        fn publishEthTransaction(data: Vec<u8>) -> String;
+        fn publishEvmTransaction(data: Vec<u8>) -> String;
         fn getAccounts() -> Vec<String>;
         fn getDatadir() -> String;
         fn getNetwork() -> String;
-        fn getEthMaxConnections() -> u32;
+        fn getEvmMaxConnections() -> u32;
         fn getDifficulty(block_hash: [u8; 32]) -> u32;
         fn getChainWork(block_hash: [u8; 32]) -> [u8; 32];
         fn getPoolTransactions() -> Vec<TransactionData>;
         fn getNativeTxSize(data: Vec<u8>) -> u64;
         fn getMinRelayTxFee() -> u64;
-        fn getEthPrivKey(key: String) -> [u8; 32];
+        fn getEvmPrivKey(key: String) -> [u8; 32];
```

Comment:
Again, EVM defines no concept of EVM address nor EVM keys. 

EVM doesn't really care for a format really. It's either ERC55, or Ethereum address format. We do follow the ethereum format. So, again, the original would be more accurate.